packages/admin/src/api/index.ts

```diff
@@ -0,0 +1,516 @@
+import type { ContextInterface } from '@textile/context'
```

Comment:
The actual API implementations, after @andrewxhill's pattern. I'm being pretty careful here about returning "nice" things to the end user. So make sure that all feels right in your review. I'm using strings for keys, because that's how we'll consume them elsewhere... but we could just as easily return the actual key classes (PrivateKey, PublicKey)...